integr8ly/integreatly-operator
# bump cro to 0.15.1

# Description

Replaces https://github.com/integr8ly/integreatly-operator/pull/663

Bump CRO to v0.15.1

## Checklist
- [x] Verified on OSD
- [x] Verified upgrade from on OSD

## Verification
- Run against cluster, ensure v0.15.1 of CRO is being used
- Run upgrade from 2.1.0, ensure v0.15.1 of CRO is being used
